internal/db/testutils.go

```
@@ -0,0 +1,31 @@
+	if err != nil {
+		t.Fatal("failed to init database: " + err.Error())
+	}
+	t.Cleanup(func() {
+		db.Close()
+	})
+	return db
+}
+
+func InsertRunningJob(t *testing.T, db *sql.DB, key string, retryCount int, mergeRequestId int64) {
+	t.Helper()
+	_, err := db.Exec(
```

golangci-lint (2.12.2)

[error] (*database/sql.DB).Exec must not be called. use (*database/sql.DB).ExecContext

(noctx)